charts/enterprise-kyverno-operator/values.yaml

```diff
@@ -57,7 +78,7 @@ kyverno:
     # -- Image repository
     repository: ghcr.io/nirmata/kyverno
     # -- Image tag
-    tag: v1.8.5-n4kbuild.1
+    tag: v1.9.2-n4k.nirmata.1
```

Comment:
Why is this different from everywhere else/`v1.9.1`?

Reply:
The 1.9.2 Chart PR is still not merged, but the image was ready. So, decided to use the 1.9.2 build directly, with 1.9.1 chart.